### PR TITLE
Create locale.xml

### PR DESCRIPTION
**Pull request by jorgeandr3s**
Spanish translation for plugins/metadata/mods34/

### CODE REVIEW

**Comment by mtub** [comment]
HI @jorgeandr3s, feel free to include the ojs plugin updates in one OJS pull request. If you want to contribute to the Spanish translation, it would be good to talk to @marcbria , see https://pkp.sfu.ca/wiki/index.php?title=OJS:_Spanish_(es_ES), because the translations follow a translation guide.

**Comment by jorgeandr3s** [comment]
Thanks @mtub I'm not sure how to merge pull request as they belong to different files. I have yet to master github. I have contacted @marcbria via email as suggested. Thanks

**Comment by marcbria** [comment]
We all have trouble with git, no worry. :-)
@jorgeandr3s , your PRs don't follow the translation guide and glossary. 
Documents sent to you.

Feel free to mail or write if you have doubts.
Amend the PRs and mention me to review before merging.

Thanks a lot for your help.

File: plugins/metadata/mods34/locale/es_ES/locale.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+<!DOCTYPE locale SYSTEM "../../../../../lib/pkp/dtd/locale.dtd">
+
+<!--
+  * plugins/metadata/mods34/locale/es_ES/locale.xml
+  *
+  * Copyright (c) 2014-2017 Simon Fraser University
+  * Copyright (c) 2003-2017 John Willinsky
+  * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+  * Localization information: https://pkp.sfu.ca/wiki/index.php?title=Translating_OxS
+  * Localization strings.
```
Comment (marcbria) [inline plugins/metadata/mods34/locale/es_ES/locale.xml]: Please, label the translation as contrib as follows:
"Localization strings (contrib)"

File: plugins/metadata/mods34/locale/es_ES/locale.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  -->
+
+<locale name="es_ES" full_name="Español (España)">
+	<message key="plugins.metadata.mods34.articleAdapter.displayName">Adaptador de artículo MODS 3.4</message>
+	<message key="plugins.metadata.mods34.articleAdapter.description">Extrae/inyecta metadatos desde y hacia los artículos.</message>
```
Comment (marcbria) [inline plugins/metadata/mods34/locale/es_ES/locale.xml]: En vez de "inyectar" (traucción literal) mejor optar por "añade".

**Comment by asmecher** [comment]
Closing these for now, as they're getting old -- @jorgeandr3s, feel free to re-open this if you're still active.

**Pull request closed without merging**